mobile-center-crashes/Crashes.js

```diff
@@ -108,14 +115,9 @@ let Helper = {
     }
 };
 
-// Android does not have "isDebuggerAttached" method
-if (Crashes && RNCrashes && RNCrashes.isDebuggerAttached) {
```

Comment:
Can you explain why removing this code?

Reply:
It's not an API we expose or document. It's in portable but works only for RN, and is not even consistent with iOS.

Reply:
(The goal was to align APIs)